src/index.js

```diff
@@ -201,3 +199,4 @@ class ContxtSdk {
 }
 
 export default ContxtSdk;
+export { toSnakeCase, toCamelCase };
```

Comment:
This should make these functions available as first class exports, ie:
`import { toSnakeCase } from '@ndustrial/contxt-sdk'`
instead of 
`import { toSnakecase } from '@ndustrial/contxt-sdk/lib/utils/objects`

as is being done in several places in nsight and contxt-app-v2

Reply:
btw, you can actually do this in one line: `export { toCamelCase, toSnakeCase } from './utils/objects';`

Reply:
oh, yea, I figured it could be done a bit simpler.